caramel/scripts/generate_ca.py

```diff
@@ -194,9 +198,14 @@ def build_ca(keyname, certname):
 
 def main():
     args = cmdline()
-    env = bootstrap(args.inifile)
+    config_uri = args.inifile
+
+    env = bootstrap(config_uri)
     settings, closer = env["registry"].settings, env["closer"]
 
+    setup_logging(config_uri)
```

Comment:
Consistent behaviour for things means that this logic should go before bootstrapping.